src/game/game.cpp

```diff
@@ -0,0 +1,40 @@
+#include "game.hpp"
+
+using namespace std;
```

Comment:
I tend to not use the `using namespace std` in order to avoid namespace clashes and explicitly see what came from the standard library. Then for longer namespaces we can use namespace alises (`namespace abc = alpha::beta::charlie`).

But this is just my preference. This is not a required change by any means. Perhaps we should look up/create some style guide and start using clang-tidy / clangformat.

Reply:
:ok_hand: